core/genesis_test.go

```diff
@@ -47,7 +47,7 @@ func TestSetupGenesis(t *testing.T) {
 	t.Skip("OVM Genesis breaks this test because it adds the OVM contracts to the state.")
 
 	var (
-		customghash = common.HexToHash("0xc4651b85bcce4003ab6ff39a969fc1589673294d4ff4ea8f052c6669aa8571a4")
```

Comment:
Reduces overall diff. Reverted to original value, doesn't matter because we skip this test anyway.